geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -45,6 +47,7 @@
 
 public class RedisSortedSet extends AbstractRedisData {
   private Object2ObjectOpenCustomHashMap<byte[], byte[]> members;
```

Comment:
Should this be Object2DoubleOpenCustomHashMap?

Reply:
This hashmap is used for quick lookup of scores associated with members. The score values are not used for sorting, they are just sent back to the client. So we're using Object2Object of byte[]'s to avoid the cost of converting to/from Doubles.

(We do convert to double for the OrderStatistics[Tree|Set], because we have to for sorting.)

Reply:
Hmm. We _could_ avoid storing the byteArray in the OrderedSetEntry, and pay the performance cost of converting from Double to byteArray for things like ZSCORE. But in this case it's probably better to spend a little memory overhead, in order to avoid every bit of performance drag we can.